avm/res/db-for-my-sql/flexible-server/tests/e2e/defaults/main.test.bicep

```diff
@@ -32,7 +36,7 @@ param namePrefix string = '#_namePrefix_#'
 // =================
 resource resourceGroup 'Microsoft.Resources/resourceGroups@2022-09-01' = {
   name: resourceGroupName
-  location: resourceLocation
+  location: enforcedLocation
```

Comment:
If added and `resourceLocation` isn't used anymore, that parameter should be removed. (same comment for other test files)

Reply:
@AlexanderSehr Updated.